compute/cloud-client/src/main/java/compute/windows/osimage/CreateImage.java

```diff
@@ -111,7 +111,7 @@ public static void createImage(String project, String zone, String sourceDiskNam
           .setImageResource(image)
           .build();
 
-      Operation response = imagesClient.insertAsync(insertImageRequest).get(3, TimeUnit.MINUTES);
+      Operation response = imagesClient.insertAsync(insertImageRequest).get(5, TimeUnit.MINUTES);
```

Comment:
Should we be showing users that they need to catch the timeoutexception and do something with it?

Reply:
I'm not sure if we catch other general exceptions and provide more information to do that for LRO. Also, LRO operation is present in most of the products and this exception is listed in: https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.longrunning

I would prefer to not add it. Please lmk your thoughts.